package/Resources/Translations/zh-Hans/Translation.ini

```
@@ -0,0 +1,1704 @@
+Client:Main:AdminRequiredRestartPrompt=您想以管理员权限重启客户端吗？; Would you like to restart the client with administrative rights?
+Client:Main:AdminRequiredTitle=需要管理员权限; Administrative privileges required
+Client:Main:AdminSuffix=(管理员); (Admin)
+Client:Main:AdvancedOptions=高级选项; Advanced Options
+Client:Main:AISpectatorsNotification=AI玩家不喜欢旁观，他们渴望玩真的！; AI players don't enjoy spectating matches. They want some action!
+Client:Main:AllGameModes=全部; All
+Client:Main:AllPlayersTab=所有玩家; All Players
+Client:Main:AttemptConnectToServer=正在尝试连接到 {0}; Attempting connection to {0}
+Client:Main:AttemptJoin=正在尝试加入游戏 {0} ...; Attempting to join game {0} ...
+Client:Main:AuthorBy=作者：; By
+Client:Main:AuthorByUnknown=作者未知; By Unknown Author
+Client:Main:AutoAllyAsANoun=自动结盟; auto ally
+Client:Main:AutoAllyingPrefix=自动结盟：; Auto Allying:
+Client:Main:AutoAllyingText1=自动结盟允许主机将起始位置分配给队伍，而不是玩家。当玩家被分配到出生点时，他们将根据这些映射自动分配到队伍。这最适合与随机队伍和随机起点一起使用。但是，只需要随机队伍。手动指定的起点将优先。; Auto allying allows the host to assign starting locations to teams, not players.@When players are assigned to spawn locations, they will be auto assigned to teams based on these mappings.@This is best used with random teams and random starts. However, only random teams is required.@Manually specified starts will take precedence.
+Client:Main:AutoAllyingTextNoTeam=禁止此位置分配给玩家。; Block this location from being assigned to a player.
+Client:Main:AutoAllyingTextRandomTeam=允许一个玩家在此，但不分配队伍。; Allow a player here, but don't assign a team.
+Client:Main:AutoAllyingTitle=自动结盟; Auto Allying
+Client:Main:AutoConnect=客户端启动时自动连接; Connect automatically on client startup
+Client:Main:AvgAILevel=平均 AI 等级：; Average AI level:
+Client:Main:Ban=封禁; Ban
+Client:Main:BanAndKickPlayer=正在封禁并将 {0} 踢出游戏...; Banning and kicking {0} from the game...
+Client:Main:Block=拉黑; Block
+Client:Main:BroadcastChannel={0} 广播频道; {0} Broadcast Channel
+Client:Main:ButtonApply=应用; Apply
+Client:Main:ButtonCancel=取消; Cancel
+Client:Main:ButtonClose=关闭; Close
+Client:Main:ButtonConnect=连接; Connect
+Client:Main:ButtonDelete=删除; Delete
+Client:Main:ButtonDownload=下载; Download
+Client:Main:ButtonGetReady=我已准备; I'm Ready
+Client:Main:ButtonIAmReady=我准备好了; I'm Ready
+Client:Main:ButtonLaunch=启动; Launch
+Client:Main:ButtonLaunchGame=启动游戏; Launch Game
+Client:Main:ButtonLoad=加载; Load
+Client:Main:ButtonLoadGame=加载游戏; Load Game
+Client:Main:ButtonLockGame=锁定游戏; Lock Game
+Client:Main:ButtonNo=否; No
+Client:Main:ButtonNotReady=未准备; Not Ready
+Client:Main:ButtonOK=确定; OK
+Client:Main:ButtonSave=保存; Save
+Client:Main:ButtonViewDownloads=查看下载; View Downloads
+Client:Main:ButtonYes=是; Yes
+Client:Main:ChangeTunnel=更换线路; Change Tunnel
```

You are using the word `线路` for tunnel while most of your translation texts use `专线`

```suggestion
Client:Main:ChangeTunnel=更换专线; Change Tunnel
```